src/containers/HomeV2/SampleCarousel.jsx

```diff
@@ -59,71 +58,66 @@ class SampleCarousel extends React.PureComponent {
         break;
       default:
         itemPerSlide = 4;
-    };
+    }
 
     return (
       <Carousel
-        slidesToShow={itemPerSlide}
```

Comment:
@Brainenjii do we need `slidesToShow` for our GUI? Without it the breakpoint of screenwidth will not work.

Reply:
I think we need it. Without this prop carousel always shows only 1 item.